<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>83</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>83</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>83</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сохранение проекта</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>13</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Путь: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="location_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browse_location_btn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Имя: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="file_name_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="save_csv_btn">
      <property name="text">
       <string>Сохранить как .csv</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_json_btn">
      <property name="text">
       <string>Сохранить как .json</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
